src/main/java/DeleteCommand.java

```diff
@@ -0,0 +1,37 @@
+import java.util.ArrayList;
+
+public class DeleteCommand extends Command{
+    static final int commandLength = 2;
+    static final int indexOfIndex = 1;
+
+    private Task taskToDeleted;
```

Comment:
taskToDeleted sounds grammatically incorrect. Maybe you meant taskToDelete or taskToBeDeleted.